pom.xml

```diff
@@ -116,16 +114,17 @@
             <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>gwt-maven-plugin</artifactId>
-                <version>${gwt.version}</version>
+                <version>2.10.0</version>
```

Comment:
Correct me if I'm wrong but this will put both GWT 2.10 (from the plugin) and 2.11 (from the project dependencies) in the gwt-compile classpath.

Reply:
In theory you can add `<dependencies>` to the plugin to handle mismatched versions, but I'm not sure how you manage away the com.google.gwt vs org.gwtproject groupId for the plugin's own classpath. That said, I haven't tried it.

Reply:
I merged #11 into this which should solve the POM issues.